lib/fastlane/plugin/xcodebuild_analyze/actions/xcodebuild_analyze_action.rb

```diff
@@ -1,46 +1,82 @@
       end
 
       def self.description
         "Run code analyzer using xcodebuild"
       end
 
       def self.authors
-        ["Meniga"]
+        ["Marcin Stepnowski"]
       end
 
-      def self.return_value
-        # If your method provides a return value, you can describe here what it does
-      end
-
-      def self.details
-        # Optional:
-        ""
+      def self.example_code
+        [
+          "
+          xcodebuild_analyze(project: 'Meniga.xcodeproj',
+                            scheme: 'Release',
+                            sdk: 'iphonesimulator'
+                            output_dir: 'tmp',
+                            output: 'html',
+                            static_analyzer: true)
+          "
+        ]
       end
 
       def self.available_options
         [
-          # FastlaneCore::ConfigItem.new(key: :your_option,
-          #                         env_name: "XCODEBUILD_ANALYZE_YOUR_OPTION",
-          #                      description: "A description of your option",
-          #                         optional: false,
-          #                             type: String)
+          FastlaneCore::ConfigItem.new(key: :workspace,
+            env_name: "XCODEBUILD_ANALYZE_WORKSPACE",
+            description: "WORKSPACE Workspace (.xcworkspace) file to use to analyze app (automatically detected in current directory)",
+            optional: true,
+            type: String),
+          FastlaneCore::ConfigItem.new(key: :project,
+            env_name: "XCODEBUILD_ANALYZE_PROJECT",
+            description: "Project (.xcodeproj) file to use to analyze app (automatically detected in current directory, overridden by --workspace option, if passed)",
+            optional: true,
+            type: String),
+          FastlaneCore::ConfigItem.new(key: :configuration,
+            env_name: "XCODEBUILD_ANALYZE_CONFIGURATION",
+            description: "Configuration used to analyze",
+            optional: true,
+            type: String),
+          FastlaneCore::ConfigItem.new(key: :scheme,
+            env_name: "XCODEBUILD_ANALYZE_SCHEME",
+            description: "Scheme used to analyze app",
+            optional: false,
+            type: String),
+          FastlaneCore::ConfigItem.new(key: :sdk,
+            env_name: "XCODEBUILD_ANALYZE_SDK",
+            description: "Use SDK as the name or path of the base SDK when building the project",
+            optional: true,
+            type: String),
+          FastlaneCore::ConfigItem.new(key: :static_analyzer,
+            env_name: "XCODEBUILD_ANALYZE_STATIC_ANALYZER",
+            description: "Run clang static analyzer if true",
+            optional: true,
+            type: Boolean),
+          FastlaneCore::ConfigItem.new(key: :output,
+            env_name: 'XCODEBUILD_ANALYZE_OUTPUT',
+            description: '- separated list of file output format (default: plist-html)',
```

Comment:
This description starts with a `-` while others do not (capital letter missing too). What would you say if we kept it consistent?